esmvaltool/cmorizers/data/formatters/datasets/ncep_ncar_r1.py

```diff
@@ -0,0 +1,169 @@
+"""ESMValTool CMORizer for NCEP-NCAR-R1 data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    https://psl.noaa.gov/data/gridded/data.ncep.reanalysis.html
+
+Last access
+    20221116
+
+Download and processing instructions
+    To facilitate the download, the links to the ftp server are provided.
+    Since the filenames are sometimes identical across different
+    save the data in two subdirectories in input_dir_path.
+    Subdirectory pressure/:
+      ftp://ftp.cdc.noaa.gov/Projects/Datasets/data.ncep.reanalysis/pressure/
+        air.mon.mean.nc
+        hgt.mon.mean.nc
+        rhum.mon.mean.nc
+        shum.mon.mean.nc
+        uwnd.mon.mean.nc
+        vwnd.mon.mean.nc
+        omega.mon.mean.nc
+      ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.dailyavgs/pressure/
+        uwnd.*.nc
+        vwnd.*.nc
+
+    Subdirectory surface/:
+      ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/surface/
+        air.mon.mean.nc
+        pr_wtr.mon.mean.nc
+        slp.mon.mean.nc
+        wspd.mon.mean.nc
+        rhum.mon.mean.nc
+      ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/surface_gauss/
+        air.2m.mon.mean.nc
+        prate.sfc.mon.mean.nc
+        tmax.2m.mon.mean.nc
+        tmin.2m.mon.mean.nc
+      ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/other_gauss/
+        tcdc.eatm.mon.mean.nc
+      ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.dailyavgs/surface_gauss/
+        prate.sft.gauss.*.nc
+      ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.dailyavgs/other_gauss/
+        ulwrf.ntat.gauss.*.nc
+
+
+    #Select the section "Pressure" and "Surface" and download the variables
+    #listed below. Since raw data on pressure levels and for surface have the
+    #same file and variable name, save the data in two different subdirectories
+    #"press" and "surf" in input_dir_path.
+
+Caveats
+
+"""
+
+import logging
+import re
+from copy import deepcopy
+from pathlib import Path
+from warnings import catch_warnings, filterwarnings
+from cf_units import Unit
+
+import iris
+from esmvalcore.cmor.table import CMOR_TABLES
+from iris import NameConstraint
+
+from esmvaltool.cmorizers.data import utilities as utils
+
+logger = logging.getLogger(__name__)
+
+
+def _fix_units(cube, definition):
+    """Fix issues with the units."""
+    """Exception is 'pr' since the units in the
+       raw file are not recoginzed correctly."""
```

Comment:
```suggestion
    """
    Fix issues with the units.
    
    Exception is `pr` since the units in the
    raw file are not recognized correctly.
    """
```